manage/CMakeLists.txt

```diff
@@ -30,4 +30,5 @@ target_link_libraries(manage PRIVATE
   model
   sndlib
   stream_audio
+  plog::plog
```

Comment:
Do you plan to create a real `log` internal library we link against instead of plog directly? It was part of my wishlist in #537 :) This lib could only be including `log.h`

Reply:
Yes, I planning to do it. In following PR I will create separate `log` submodule with some functionality from ddebug and misc.